catalystwan/models/configuration/feature_profile/sdwan/policy_object/security/aip.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Cisco Systems, Inc. and its affiliates
+
+from typing import Literal
+
+from pydantic import AliasPath, Field
+
+from catalystwan.api.configuration_groups.parcel import Global, _ParcelBase
+from catalystwan.models.configuration.feature_profile.sdwan.transport.bgp import RefIdItem
+
+TlsDecryptionAction = Literal["decrypt", "neverDecrypt", "skipDecrypt"]
+
+
+class AdvancedInspectionProfileParcel(_ParcelBase):
+    type_: Literal["unified/advanced-inspection-profile"] = Field(
+        default="unified/advanced-inspection-profile", exclude=True
+    )
+    description: str = ""
```

Comment:
no need to override description from _ParcelBase
```suggestion
```

Reply:
For this parcel description is required. I think it would be convenient for users to keep it optional in sdk.